packages/jest-watcher/src/lib/Prompt.ts

```diff
@@ -8,47 +8,29 @@
 import {ScrollOptions} from '../types';
 import {KEYS} from '../constants';
 
-export default class Prompt {
-  _entering: boolean;
-  _value: string;
-  _onChange: () => void;
-  _onSuccess: (value?: string) => void;
-  _onCancel: (value?: string) => void;
-  _offset: number;
-  _promptLength: number;
-  _selection: string | null;
-
-  constructor() {
-    // Copied from `enter` to satisfy TS
-    this._entering = true;
-    this._value = '';
-    this._selection = null;
-    this._offset = -1;
-    this._promptLength = 0;
-
-    this._onChange = () => {};
-    this._onSuccess = () => {};
-    this._onCancel = () => {};
+const noop = () => {};
 
-    this._onResize = this._onResize.bind(this);
-  }
+export default class Prompt {
+  _entering: boolean = true;
+  _value: string = '';
+  _onChange: () => void = noop;
+  _onSuccess: (value?: string) => void = noop;
+  _onCancel: (value?: string) => void = noop;
+  _offset: number = -1;
+  _promptLength: number = 0;
+  _selection: string | null = null;
 
-  private _onResize() {
+  private _onResize = () => {
     this._onChange();
-  }
+  };
 
   enter(
     onChange: (pattern: string, options: ScrollOptions) => void,
     onSuccess: () => void,
     onCancel: () => void,
   ) {
-    this._entering = true;
```

Comment:
these should be kept, since we need to reset the values in `enter`

Reply:
Moved to `_resetState` function → https://github.com/facebook/jest/pull/7881/commits/68ce8b49e7dcb54eb5e5d75fa80d5cfac7426fa3
and added a test for that.

I changed this code and nothing didn't tell me that I can't, seems like "a lack of coverage", doesn't it? WDYT?

Reply:
yeah, probably need to expand an e2e test to activate a plugin, type something, go out of the plugin, go back in, then assert that the prompt is empty

Reply:
So, do you think it should be an e2e test? (I've added a unit one, probably doesn't make sense then)

Reply:
I guess it's better to address separately.